plex_set_tracks.py

```diff
@@ -455,15 +432,13 @@ def printSubtitles(streams, startIndex=1):
             streams(list<:class:`~plexapi.media.SubtitleStream`>): SubtitleStreams to be printsd
             startIndex(int): Index to start at (default = 1)
     """
-    count = startIndex
-    for stream in streams:
+    for i, stream in enumerate(streams):
```

Comment:
Similarly, this loop would be more clear if you just started from the startIndex.

`for i, stream in enumerate(streams, startIndex)`